File: app/src/main/java/com/example/coronatracker/Api/methods.java
package com.example.coronatracker.Api;

import com.example.coronatracker.DataClasses.Root;
import com.example.coronatracker.DataClasses.indiaContactModel.stateContacts;
import com.example.coronatracker.DataClasses.indiaModel.indiaStates;
import com.example.coronatracker.DataClasses.world;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;


public interface methods {
    @GET("covid-19/countries")
    Call<List<Root>> getData();

    @GET("all?yesterday")
    Call<world> getWorld();

    @GET("covid19-in/contacts")
    Call<stateContacts> getContacts();

    @GET("covid19-in/stats/latest")
    Call<indiaStates> getStates();
}

File: app/src/main/java/com/example/coronatracker/api/methods.kt
package com.example.coronatracker.api

import com.example.coronatracker.dataClasses.Root
import retrofit2.http.GET
import com.example.coronatracker.dataClasses.world
import com.example.coronatracker.dataClasses.indiaContactModel.stateContacts
import com.example.coronatracker.dataClasses.indiaModel.indiaStates
import retrofit2.Call

interface methods {
    @get:GET("covid-19/countries")
    val data: Call<List<Root?>>

    @get:GET("all?yesterday")
    val world: Call<world?>?

    @get:GET("covid19-in/contacts")
    val contacts: Call<stateContacts?>?

    @get:GET("covid19-in/stats/latest")
    val states: Call<indiaStates?>?
}